src/components/colorbar/draw.js

```diff
@@ -253,10 +254,10 @@ module.exports = function draw(gd, id) {
 
         cbAxisOut._axislayer = container.select('.cbaxis');
         var titleHeight = 0;
-        if(['top','bottom'].indexOf(opts.titleside)!==-1) {
+        if(['top', 'bottom'].indexOf(opts.titleside) !==- 1) {
```

Comment:
Looks like the space was a bit too eager there.

Should be `if(['top', 'bottom'].indexOf(opts.titleside) !== -1) {`